roles/container_runtime/tasks/docker_local_volume_setup.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create file system on local volume device
+  filesystem:
+    fstype: "{{ container_runtime_docker_local_volume_fstype | default('xfs') }}"
+    dev: "{{ container_runtime_docker_local_volume_device }}"
+  when:
+  - container_runtime_docker_local_volume_device is defined
+
+- name: Create mount entry for local volume
+  mount:
+    path: /var/lib/origin/openshift.local.volumes
```

Comment:
Everything in this task should be configurable; we should abstract this to a dictionary (or a list of dictionaries and use with_items) instead of a variable for each.

In fact, a list of dictionaries would allow us to specify [] by default and do nothing, forcing the user to define exactly what they want and where they want it.

Reply:
@michaelgugino I like the list of dictionaries idea; We can abstract this further so any extra block device may be formatted (or not) and assigned to any path on the file system.

This will allow us to handle both local_volume storage and etcd volume storage.